pkg/manager/member/tidb_member_manager_test.go

```diff
@@ -742,6 +743,33 @@ func TestTiDBMemberManagerSyncTidbService(t *testing.T) {
 				g.Expect(svc.Spec.Ports[0].Name).To(Equal("mysql-tidb"))
 			},
 		},
+		{
+			name: "Update service should remain healthcheck node port",
+			prepare: func(tc *v1alpha1.TidbCluster, indexers *fakeIndexers) {
+				tc.Spec.TiDB.Service = &v1alpha1.TiDBServiceSpec{
+					ServiceSpec: v1alpha1.ServiceSpec{
+						Type: corev1.ServiceTypeLoadBalancer,
+					},
+					ExternalTrafficPolicy: &policyLocal,
+				}
+				_ = indexers.svc.Add(&corev1.Service{
```

Comment:
`_ =` is required by `linter`?

Reply:
I have not tried, here I ignore error explicitly in favor of clarity